ONSdigital/eq-author-app
# Add title to list collector pages

> ### BEFORE MAKING YOUR PR
>
> Please ensure:
>
> - There are no linting errors, all tests must pass
> - PR is named after JIRA ticket number e.g. EAR-###
> - **Accesibility** checks are completed:
>   - Elements have discernible and consistent focus states
>   - Elements can be navigated to and used by just a keyboard
>   - Elements and text can be read out by a screen reader, and the descriptions are understandable
> - Your feature / bug fix works across **GCP** and **AWS** (where appropriate)
>   - Are modifications to eq-publisher-v3 required?
>   - Are modifications to the Firestore data source required?

---

### What is the context of this PR?

Adds title to list collector pages to prevent crash
Adds empty string for undefined option labels to display validation errors

### How to review

- On `master` branch, import crashing "Example questionnaire"
- Check that this questionnaire crashes when opened locally
- Switch to branch `add-title-to-list-collector-pages` (this branch)
- Check that this questionnaire does not crash when opened locally
- Check that the qualifier page's empty option labels display validation errors

### What to do after everything is green

1. - [ ] Bring this branch up-to-date with Origin/Master
2. - [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. - [ ] Click the **Merge** button on this pull request
4. - [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. - [ ] Move the Jira ticket for this task into the next stage of the process
